templates/beta/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   devnet:
     image: aragon/geth-devnet
     volumes:
-      - ${ETHDATA_LOCAL_PATH}:/root/.ethdata
+      - ${ETHDATA_LOCAL_PATH}:/root/.ethereum
```

Comment:
Shall we set a user in aragon/geth-devnet, instead of running geth as root?

Reply:
Yes, but that image (`aragon/geth-devnet`) Dockerfile is not in this repo, right? So that change needs to be made somewhere else.

Reply:
Yes sorry, I was talking about https://github.com/aragon/geth-devnet

Reply:
So maybe we can just merge this one for now and leave that change for another PR

Reply:
Yes 👌